test/source-map/test-api.js

```diff
@@ -13,8 +13,8 @@ define(function (require, exports, module) {
   try {
     sourceMap = require('../../lib/source-map');
   } catch (e) {
-    sourceMap = {};
-    Components.utils.import('resource:///modules/devtools/SourceMap.jsm', sourceMap);
+    var {devtools} = Components.utils.import("resource://gre/modules/devtools/Loader.jsm", {});
+    sourceMap = devtools.require("devtools/toolkit/sourcemap/source-map");
```

Comment:
These tests have to be able to run in node, so no ES6 :(

https://travis-ci.org/mozilla/source-map/jobs/73406398#L107